x/bounty/types/genesis.go

```diff
@@ -21,6 +24,31 @@ func DefaultGenesisState() *GenesisState {
 
 // ValidateGenesis - validate bounty genesis data
 func ValidateGenesis(data *GenesisState) error {
-	// TODO: implement ValidateGenesis
+	programIds := make(map[uint64]bool)
+	for _, program := range data.Programs {
+		if program.ProgramId > data.StartingProgramId {
+			return fmt.Errorf("error program id")
+		}
+
+		_, ok := programIds[program.ProgramId]
+		if ok {
+			//repeat program
+			return fmt.Errorf("repeat programId:%d", program.ProgramId)
+		}
+		programIds[program.ProgramId] = true
+	}
+
+	for _, finding := range data.Findings {
```

Comment:
should call validate functions for each of them

Reply:
fixed